### PR TITLE
Allow URL property in uploader to be a string or string returning function

### PR DESCRIPTION
**Pull request by strongui**
Fixes #

### CODE REVIEW

**Comment by xdan** [comment]
@strongui Hi, please remove from your PR - `build` folder.

File: src/modules/uploader/uploader.ts [hunk 1]
```diff
@@ -269,7 +269,10 @@ export class Uploader extends ViewComponent implements IUploader {
 					},
 					method: this.o.method || 'POST',
 					data: request,
-					url: this.o.url,
+					url:
+						typeof this.o.url === 'function'
```
Comment (xdan) [inline src/modules/uploader/uploader.ts]: Jodit has helper `src/core/helpers/checker/is-function`
```js
if (isFunction(this.o.url))
```

Comment (strongui) [reply src/modules/uploader/uploader.ts]: @xdan Thank you, I have changed the code to use the method.

**Comment by strongui** [comment]
> 
> 
> @strongui Hi, please remove from your PR - `build` folder.

I have deleted the build folder

**Pull request merged**